extensions/http-curl/tests/HttpGetIntegrationTest.cpp

```diff
@@ -74,6 +71,8 @@ class HttpResponder : public CivetHandler {
 };
 
 int main(int argc, char **argv) {
+  using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
```

Comment:
I'm not sure this has anything to do in utils in case we only use it in for verification. Should be part of a test namespace.

Reply:
I don't think we have a test namespace. Some test utility are in `utils` and some are in global scope.